src/components/typography/typography.spec.tsx

```diff
@@ -47,6 +49,20 @@ const pStyling = {
   listStyleType: undefined,
 };
 
+function render(
+  ui: React.ReactElement,
+  options?: RTLRenderOptions,
+  document = globalThis.document
+) {
+  if (document.body.innerHTML.length >= 1) {
+    throw new Error(
+      "Found DOM to be non-empty before rendering. Please make sure to call cleanup() after each test."
+    );
+  }
```

Comment:
**thought**: it might be worth us writing a custom render function for cleaning up React Portals when we properly start migrating tests over to RTL. Just so we don't have to duplicate this in several test files.

Reply:
Removed all the ref/tab-index work from the PR so this won't be required here but it would be worth doing sooner or later, yeah.